catalog/core/catalog-core-backupplugin/src/main/java/ddf/catalog/backup/CatalogBackupPlugin.java

```diff
@@ -126,10 +126,11 @@ public void shutdown() {
         // The JVM will throw an Interrupted Exception.
       } catch (InterruptedException e) {
         LOGGER.warn("Backup of metacards interrupted. Some metacards might not be backed up.");
+        Thread.currentThread().interrupt();
         throw new IllegalStateException(e);
       }
       final List<Runnable> failures = executor.shutdownNow();
-      if (failures.size() > 0) {
+      if (failures.isEmpty()) {
```

Comment:
❗️ It looks like this should be `!failures.isEmpty()`, unless I'm missing something.

Reply:
No you're not. Good catch!